src/app.html

```diff
@@ -3,9 +3,9 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    %svelte.head%
+    %sveltekit.head%
   </head>
   <body>
-    <div id="app">%svelte.body%</div>
+    <div id="app">%sveltekit.body%</div>
```

Comment:
ooh is this prototype in svelte!? that's awesome, I've been wanting to try a svelte project for ages.